### PR TITLE
added windows build instructions

### PR DESCRIPTION
**Pull request by scopatz**
I always forget how to do this, so I thought this would be a helpful reference.

### CODE REVIEW

File: src/buildwin.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+------------------------------
+Some projects provide hooks for CMake to build the project. The following
+example ``bld.bat`` file demonstrates how to build a traditional, out-of-core
+build for such projects.
+
+**CMake-based bld.bat:**
+
+.. code-block:: bat
+
+    setlocal EnableDelayedExpansion
```
Comment (jakirkham) [inline src/buildwin.rst]: I don't think we generally need this. @msarahan @mingwandroid , thoughts?

**Comment by scopatz** [comment]
Thanks for the feedback, I have updated the PR to reflect the comments.

File: src/buildwin.rst (hunk 1 above)
Comment (msarahan) [inline src/buildwin.rst]: Better to keep it, IMHO. It leads to more intuitive behavior.

Comment (scopatz) [inline src/buildwin.rst]: Added it back, and I agree.

Comment (jakirkham) [inline src/buildwin.rst]: Ok. Should we add `@echo off` too?

Comment (scopatz) [inline src/buildwin.rst]: I don't think it matters much, but my inclination is to have echo on, like with bash and `set -x`

**Pull request merged**